apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/preadj/McsClaimResponseTransformerV2.java

```diff
@@ -34,11 +48,73 @@ static ClaimResponse transform(MetricRegistry metricRegistry, Object claimEntity
    * @return a FHIR {@link ClaimResponse} resource that represents the specified {@link
    *     PreAdjMcsClaim}
    */
-  private static ClaimResponse transformClaim(Object claimGroup) {
+  private static ClaimResponse transformClaim(PreAdjMcsClaim claimGroup) {
     ClaimResponse claim = new ClaimResponse();
 
-    // TODO: [DCGEO-98] Transform claim
+    claim.setId("m-" + claimGroup.getIdrClmHdIcn());
+    claim.setMeta(new Meta().setLastUpdated(Date.from(claimGroup.getLastUpdated())));
+    claim.setExtension(getExtension(claimGroup));
+    claim.setIdentifier(getIdentifier(claimGroup));
+    claim.setStatus(ClaimResponse.ClaimResponseStatus.ACTIVE);
+    claim.setOutcome(ClaimResponse.RemittanceOutcome.QUEUED);
+    claim.setType(getType());
+    claim.setUse(ClaimResponse.Use.CLAIM);
+    claim.setCreated(new Date());
+    claim.setInsurer(new Reference().setIdentifier(new Identifier().setValue("CMS")));
+    claim.setPatient(getPatient(claimGroup));
+    claim.setRequest(new Reference(String.format("Claim/m-%s", claimGroup.getIdrClmHdIcn())));
 
     return claim;
   }
+
+  private static List<Extension> getExtension(PreAdjMcsClaim claimGroup) {
+    return Collections.singletonList(
+        new Extension()
+            .setUrl("https://dcgeo.cms.gov/resources/variables/mcs-status")
+            .setValue(
+                new Coding(
+                    "https://dcgeo.cms.gov/resources/variables/mcs-status",
+                    "" + claimGroup.getIdrStatusCode(),
```

Comment:
`idrStatusCode` is a String already.  Is the extra quote in case it's a null?

Reply:
Copy paste error, the FISS status code used a `char`